modules/services/remmina.nix

```diff
@@ -0,0 +1,72 @@
+{ config, lib, pkgs, ... }:
+
+let cfg = config.services.remmina;
+in {
+  options.services.remmina = {
+    enable = lib.mkEnableOption "Remmina";
+
+    package = lib.mkPackageOption pkgs "remmina" { };
+
+    addRdpMimeTypeAssoc = lib.mkEnableOption "Remmina RDP file open option" // {
+      default = true;
+    };
+
+    systemdService = {
+      enable = lib.mkEnableOption "Systemd Remmina service" // {
+        default = true;
+      };
+
+      startupFlags = lib.mkOption {
+        type = with lib.types; listOf str;
+        default = [ "--icon" ];
+        description = ''
+          Startup flags documented in the manpage to run at service startup.
+        '';
+      };
+    };
+  };
+
+  config = lib.mkIf cfg.enable (lib.mkMerge [
+    { home.packages = [ cfg.package ]; }
+
+    (lib.mkIf cfg.systemdService.enable {
+      systemd.user.services.remmina = {
+        Unit = {
+          Description = "Remmina remote desktop client";
+          Documentation = "man:remmina(1)";
+          Requires = [ "graphical-session-pre.target" ];
+        };
+
+        Service = {
+          Type = "simple";
+          ExecStart = "${lib.getExe cfg.package} ${
+              lib.concatStringsSep " " cfg.systemdService.startupFlags
```

Comment:
```suggestion
              escapeShellArgs cfg.systemdService.startupFlags
```

Reply:
Does it work properly with `escapeShellArgs`? I've had a lot of issues getting that to work thanks to how Systemd parses it

Reply:
replaced in d371fc9